### DISCUSSION TITLE
Background Color für bsCookie Message

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#558] [Q&A]
I tried to install your bS Cookie plugin in accordance with your installation guide https://bootscore.me/documentation/plugin/bs-cookie-settings/ . Everything went well. However, I didn't find a way to configure the overlay dialog via CSS. Especially, I need an option to set the background color, because - out of the box - the dialog uses 'transparent' Do you know such a way ?

**Comment by kreincke** [comment]
Ok, I've found a very 'brutal' = hard coded method: we can add a section in our _bscore_custom.scss like this

#c-inr { 
  border-color: green; 
  border-width: 5px; 
  border-style: solid; 
  background-color: #fff; 
  color:#00ff00;  
  padding: 10px;
}

Do you know a more elegant way? The down-side of this way is that it depends on the id 'c-inr' ...

Many thanks for your answers... heartfully KR

**Reply by crftwrk** [reply]
> Do you know a more elegant way?

What's wrong there? Overrides are the best way to customise plugin.

You can edit source js https://github.com/bootscore/bs-cookie-settings/blob/main/src/js/cookie-settings.js and add/change classes. But this will be overridden by the next update and is tricky. So, your solution is the best choice.

Basti

**Comment by kreincke** [comment]
It's very good to read that this is your canonical way. There's nothing wrong with that solution. The only 'downside' is that it depends on the IDs 'c-inr' and successors because you as the developer must keep in mind that a change on your side (code) could destroy the work of your users. But if you say, that this solution is your preferred way, then I am sure that you will keep it in mind.

Many thanks for answering my question so immediately -  a pleasure as the last time.
KR

**Reply by crftwrk** [reply]
If you want, you can add classes to those id‘s in the source and we can merge them here.
